### PR TITLE
Remove unused stressbench

### PR DESCRIPTION
**Pull request by apc999**
### What changes are proposed in this pull request?

Remove unused stressbench

### Why are the changes needed?

With architecture after 300, a set of stressbench are no longer relevant. 

### Does this PR introduce any user facing changes?

Removed StressJobServiceBench, RpcBench, StressMasterBench, MaxFileBench, GetPinnedFileIdsBench, CompactionBench and FuseIOBench

### CODE REVIEW

File: dora/core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java [hunk 1]
```diff
@@ -1,1111 +0,0 @@
...
-import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.stream.Collectors;
-
-/**
- * Unit tests for {@link BlockMaster}.
- */
-public class BlockMasterTest {
-  public static final long CAPACITY = 20L * 1024 * 1024 * 1024; // 20GB
```
Comment (jiacheliu3) [inline dora/core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java:92]: deleting this test belongs to a separate PR?

File: dora/tests/pom.xml [hunk 2]
```diff
@@ -209,6 +209,13 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```
Comment (jiacheliu3) [inline dora/tests/pom.xml:212]: is this intented?

File: dora/core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java (hunk 1 above)
Comment (apc999) [reply dora/core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java]: good call. removed this deletion from this PR

File: dora/tests/pom.xml (hunk 2 above)
Comment (apc999) [reply dora/tests/pom.xml]: removed

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, merge this please
